### PR TITLE
modify_the_HARAKIRI value

### PR DESCRIPTION
**Pull request by Basma1912**

### CODE REVIEW

File: bin/uwsgi.ini [hunk 1]
```diff
@@ -7,7 +7,7 @@ module = nucleus.wsgi:application
 
 die-on-term = true  ; Shutdown on SIGTERM (default is respawn)
 enable-threads = true  ; Support multithreading
-harakiri = 30  ; Kill hung requests after 30 seconds
+harakiri = 300  ; Kill hung requests after 30 seconds
```
Comment (pmac) [inline bin/uwsgi.ini:10]: I hate to be pedantic, but would you mind also updating the comment?

Comment (pmac) [reply bin/uwsgi.ini:10]: Also, is 5min too much? Would it be better to just bump it to 60 or 90s first?

Comment (Basma1912) [reply bin/uwsgi.ini:10]: sure

**Review by pmac — APPROVED** [review]
Thank you!

**Pull request merged**